src/webcomponents/job/job-monitor.js

```diff
@@ -42,214 +41,209 @@ export class JobMonitor extends LitElement {
-        this.jobs = newList.map((job, i) => {
-            if (i < k) {
-                // handle the new jobs
-                NotificationUtils.dispatch(this, NotificationUtils.NOTIFY_INFO, {
-                    message: `${job.id}, "The job has been added`,
-                });
-                return {...job, updated: true};
-            } else {
-                // handle the change of state
-                // FIXME remove this in v2.3
-                const statusId = job.internal.status.id || job.internal.status.name;
-                const oldStatusId = oldList[i - k].internal.status.id || oldList[i - k].internal.status.name;
-                if (statusId !== oldStatusId) {
-                    NotificationUtils.dispatch(this, NotificationUtils.NOTIFY_INFO, {
-                        message: `${job.id} The job has now status ${job?.internal?.status?.id || job?.internal?.status?.name}`,
-                    });
-                    return {...job, updated: true};
-                } else {
-                    // if the ids are the same I want to keep the `updated` status
-                    // return {...job, updated: false};
-                    return {...oldList[i - k]};
-                }
+    launchMonitor() {
+        clearInterval(this._interval);
+        if (this.opencgaSession) {
+            // Check if the user has VIEW_JOBS permission in the current study
+            if (CatalogUtils.checkPermissions(this.opencgaSession.study, this.opencgaSession.user.id, "VIEW_JOBS")) {
+                this.fetchLastJobs();
+                this._interval = setInterval(() => this.fetchLastJobs(), this._config.interval);
             }
-        });
-        // accumulate all the updated (not visited) status
-        this.updatedCnt = this.jobs.reduce((acc, job) => job.updated && !job._visited ? acc + 1 : acc, 0);
-        this.requestUpdate();
-        await this.updateComplete;
-        this._jobs = this.jobs;
+        }
     }
 
     fetchLastJobs() {
-        if (!this?.opencgaSession?.token || !$("#job-monitor").is(":visible")) {
-            clearInterval(this.interval);
-            return;
-        }
-
-        const query = {
-            study: this.opencgaSession.study.fqn,
-            internalStatus: "PENDING,QUEUED,RUNNING,DONE,ERROR,ABORTED",
-            limit: this._config.limit || 10,
-            sort: "creationDate",
-            include: "id,internal.status,tool,creationDate",
-            order: -1
-        };
-        this.opencgaSession.opencgaClient.jobs().search(query)
-            .then(async restResponse => {
-                // console.log("restResponse", restResponse);
-                // first call
-                if (!this._jobs.length) {
-                    this._jobs = restResponse.getResults();
+        this.opencgaSession.opencgaClient.jobs()
+            .search({
+                study: this.opencgaSession.study.fqn,
+                internalStatus: "PENDING,QUEUED,RUNNING,DONE,ERROR,ABORTED",
```

Comment:
Two comments here:
1. The job status UNREGISTERED is not queried here, but is modelled in this.JOB_TYPES. Should it be included here?
2. Would it be worth to consider the full list of possible statuses modelled in OpenCGA or has been previously decided to displayed and labelled a subset of them? See link below:
https://github.com/opencb/opencga/blob/develop/opencga-core/src/main/java/org/opencb/opencga/core/models/common/Enums.java#L370-L371

Reply:
Noted to discussion with Nacho. In the meanwhile, I have removed the `UNREGISTERED` from the list of types associated with the Finished category.